tests/state/units/modules/defineunits/__init__.py

```diff
@@ -0,0 +1,70 @@
+
+def add5m(state):
```

Comment:
Why is this called `add 5m`?

Reply:
Ty. I've adapted the name to reflect the functionality.